tests/helpers/projects.ts

```diff
@@ -15,6 +15,9 @@ const openProjectCreationForm = async (page: Page) => {
 
   await expect(page.getByText("Project name")).toBeVisible();
   await expect(page.getByText("Loading storage pools")).not.toBeVisible();
+
+  //Set default profile network to None
+  await page.getByLabel("Default profile network").selectOption("No network");
```

Comment:
```suggestion
  await page.getByLabel("Default profile network").selectOption("No network");
```
I think the comment is redundant considering the line of code below, wdyt?